Python 3.6 compatibility and upgrade ONIX XSDs to 3.0.6+cl45

jonny5532 opened the pull request:
I've replaced the `os.getcwdu()` call with the `six` version for Python3 compatibility (now works in Python 3.6) and added the dependency to `setup.py`.

Have also updated the ONIX 3.0 XSDs to the latest from https://www.editeur.org/93/Release-3.0-Downloads/

jonny5532 commented:
Took a few goes to get the tox/Travis configuration to work (they have both changed a bit).

Why are there 'nocover' versions of the tests? What is gained by not running/displaying the coverage?

titusz commented:
When I created the repository I used some default templates for the tox/travis scaffolding without really understanding how it works. I guess we can remove the 'nocover' version to speed up tests. I will check your pull request soon and if things are fine I will merge and prepare a new release. Big thanks for your contribution!

jonny5532 commented:
Cool - I'm not very familiar with the Travis/tox config myself but I hope I've made a reasonable job of it - I've removed all the nocover tests (and also updated the appveyor config which had one also).

(and thanks for the app, it is very useful!)
